spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe HomeController, :type => :controller do
+
+  describe "GET #index" do
+    it "returns HTTP status 200" do
+      get :index
+      expect(response).to have_http_status(200)
```

Comment:
Tá certo assim, mas outra forma de fazer é esperar que a resposta seja bem-sucedida, acho que fica mais fácil de ler que usando os status

`expect(response).to be_sucessful`